packages/web/src/app/proposal/page.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+
+import React from "react";
+// import { useState } from "react";
+
+import FlexWrapper from "@sparcs-students/web/common/components/FlexWrapper";
+import Typography from "@sparcs-students/web/common/components/Typography";
+import SearchThreeInput from "@sparcs-students/web/features/proposal/components/SearchThreeInput";
+import Button from "@sparcs-students/web/common/components/Buttons/Button";
+import ViewResult from "@sparcs-students/web/features/proposal/components/ViewResult";
+import IncomeTable from "@sparcs-students/web/features/proposal/components/IncomeTable";
+import ExpenditureTable from "@sparcs-students/web/features/proposal/components/ExpenditureTable";
+import TotalTable from "@sparcs-students/web/features/proposal/components/TotalTable";
+import {
+  mockExpenditureData,
+  mockIncomeData,
+  mockTotalData,
+  mockViewResultData,
+} from "@sparcs-students/web/features/proposal/services/_mock/mockProposalTableData";
+import PageTitle from "@sparcs-students/web/common/components/PageTitle";
+
+const Proposal = () => (
+  // const [semester, setSemester] = useState<string>(""); // 2022~2028년, 봄학기/가을학기
+  // const [documentType, setDocumentType] = useState<string>(""); // 예산안, 결산안, 사업보고서, 사업계획서
+  // const [organization, setOrganization] = useState<string>("");
+
+  <FlexWrapper direction="column" gap={16}>
+    <PageTitle>예결산 조회</PageTitle>
+    <FlexWrapper direction="column" gap={60} style={{ padding: "32 0px" }}>
+      <FlexWrapper direction="column" gap={32} style={{ padding: "0 100px" }}>
+        <FlexWrapper direction="column" gap={16}>
+          <Typography fs={24} lh={30} color="BLACK" fw="SEMIBOLD">
+            조회 가이드
+          </Typography>
+          <SearchThreeInput />
+        </FlexWrapper>
+        <FlexWrapper direction="row" gap={8}>
+          <Button buttonText="조회" style={{ marginLeft: "auto" }} />
+        </FlexWrapper>
+      </FlexWrapper>
+      <FlexWrapper direction="column" gap={32} style={{ padding: "0 100px" }}>
```

Comment:
이건 전적으로 제 잘못인데 이미 페이지에 기본적으로 반응형에 따른 padding 설정이 돼 있어서 여기서 padding: "0 100px"를 빼도 될 거에요. 그러면 테이블이 좀 더 가로로 길어짐서 한 눈에 들어오는 형태가 될 거에요.
그래도 지금 테이블에 가로 스크롤 생기면 안 되고, 코드 부분 열이 너무 넓은 건 해결을 해야 할 것 같습니다..!
```suggestion
      <FlexWrapper direction="column" gap={32}>
```

Reply:
![image](https://github.com/user-attachments/assets/7f61247b-c78f-40d0-855e-cfd5f6a48b2a)
그부분 패딩을 빼면 이렇게 되어버려요..

Reply:
오마이.... 알겠습니다ㅠ 이거는 제가 디자인 고친걸 빠른시일내에 드릴게요